stable/telegraf/values.yaml

```
@@ -32,6 +32,11 @@ service:
   type: ClusterIP
   annotations: {}
 
+  ## Creates a service account for this telegraf deployment and adds the given array of rules
+  account:
+    enabled: false
+    rules: []
```

Thanks for the PR, creating an SA is a good idea indeed
As per link provided by@davidkarlsen 
```suggestion
rbac:
  # Specifies whether RBAC resources should be created
  create: true

serviceAccount:
  # Specifies whether a ServiceAccount should be created
  create: true
  # The name of the ServiceAccount to use.
  # If not set and create is true, a name is generated using the fullname template
  name:
```